README.md

```diff
@@ -70,8 +70,23 @@ Note: Currently the Dashboard does not have complete coverage of Apache APISIX f
 
 Support the following ways currently.
 
-- [Source Codes](./docs/deploy.md)
-- [Docker](./docs/deploy-with-docker.md)
+1. [Source Codes](./docs/deploy.md)
+2. [Docker](./docs/deploy-with-docker.md)
+3. RPM package（only for CentOS 7）
+   - install runtime dependencies: [etcd](https://etcd.io/docs/v3.4.0/dl-build/) 3.4.0+
```

Comment:
Please use a new file called `deploy-with-rpm.md` to have detailed description.

Reply:
fixed.